packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/order/translations/Messages_fr_FR.json

```diff
@@ -74,5 +74,8 @@
   "telephony_line_phone_detail_link": "Fiche technique",
   "telephony_line_phone_order_validate": "Valider ma commande",
   "ovh_contracts_agree": "Je reconnais avoir pris connaissance des contrats ci-dessus.",
-  "telephony_line_phone_choice_fees_description": "{{feesText}} : {{feesPriceText}}"
+  "telephony_line_phone_choice_fees_description": "{{feesText}} : {{feesPriceText}}",
```

Comment:
there is nothing to translate here, you could simply add the ":" in your template